src/objects.c

```diff
@@ -37,7 +37,7 @@ struct p11prov_obj {
     } data;
 
     CK_ATTRIBUTE *attrs;
-    unsigned long numattrs;
+    int numattrs;
```

Comment:
Why this change?

Reply:
because of p11prov_move_alloc_attrs(), it takes an int, and we really do not need unsigned long many attrs :-)